constants/ng-constants.json

```diff
@@ -131,8 +131,7 @@
         "CANCELLED": "CANCELLED",
         "AWAITING": "AWAITING"
       },
-      "addAppointmentUrl" : "#/home/manage/appointments/calendar/new",
-      "editAppointmentUrl" : "#/home/manage/appointments/calendar/edit"
+      "appointmentsV2Url" : "/appointments-v2/#/home/manage/appointments"
```

Comment:
@AlekhyaYalla we might have to check with @bsneha90 that **appointmentsV2Url** is going to be the final URL for appointments. We can still change this later.